Bottom: 4ff33530d74623d8ed8d9812dc22f5c48b020c63
Top:    0b6c8394143f9ed6f4cf36065b199ca8e6563d3e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 00:29:01 +0100

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/hid.h b/src/hid.h
index ee48b9f..7966346 100644
--- a/src/hid.h
+++ b/src/hid.h
@@ -214,6 +214,7 @@ extern "C"
 #define SL_INVISIBLE	0x0060
 #define SL_FAB		0x0070
 #define SL_ASSY		0x0080
+#define SL_RATS		0x0090
 /* Callers should use this.  */
 #define SL(type,side) (~0xfff | SL_##type | SL_##side##_SIDE)
